### ISSUE TITLE
Keras step_function doesn't return accuracy

### ISSUE THREAD

**Issue by Esther2013** [SciSharp/TensorFlow.NET#630]
Keras step_function only returns loss value, should return accuracy as well.

![image](https://user-images.githubusercontent.com/33488376/97816568-c2c19400-1c5b-11eb-94e8-31c34625dfa1.png)

Below is python version:

![image](https://user-images.githubusercontent.com/33488376/97816597-f3093280-1c5b-11eb-824d-9a31a5df2bae.png)

**Comment by Oceania2018** [comment]
![image](https://user-images.githubusercontent.com/1705364/98486106-ff076e00-21e0-11eb-960f-eaa88c6ee5b9.png)
